src/mono/System.Private.CoreLib/src/System/Reflection/Emit/MethodBuilder.Mono.cs

```diff
@@ -351,7 +351,7 @@ internal void check_override()
                 foreach (MethodInfo m in override_methods)
                 {
                     if (m.IsVirtual && !IsVirtual)
-                        throw new TypeLoadException(string.Format("Method '{0}' override '{1}' but it is not virtual", name, m));
+                        throw new TypeLoadException(SR.Format("Method '{0}' override '{1}' but it is not virtual", name, m));
```

Comment:
This should either use `string.Format` or the message should be moved into `Strings.resx`.